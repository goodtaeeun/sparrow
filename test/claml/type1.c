enum a { b };
struct c {
  enum a d;
};
typedef c;
struct c a;
int main() { return a.d; }
