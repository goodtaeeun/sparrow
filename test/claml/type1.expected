/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "type1.c"
enum a { b = 0 };
#line 2 "type1.c"
struct c {
  enum a d;
};
#line 5 "type1.c"
typedef int c;
#line 6
struct c a;
#line 7 "type1.c"
int main(void) {

  {
#line 7
    return ((int)a.d);
  }
}
